CONTRIBUTING.md

```diff
@@ -1,61 +1,520 @@
 # Contributing Guidelines
 
-Thank you for your interest in contributing to our project. Whether it's a bug report, new feature, correction, or additional
-documentation, we greatly value feedback and contributions from our community.
+Thank you for your interest in contributing to the Android Amplify
+framework. Whether it's a bug report, new feature, correction, or
```

Comment:
"Amplify Android is part of the Amplify Framework" or something like that

Reply:
Alternatively: "Thank you for your interest in contributing to the Android Library of the Amplify Framework."

Reply:
I addressed this in three ways.

1. I went with:
> "Thank you for contributing to the Android distribution of the Amplify framework."

2. Also capitalized all uses of `Framework`, to codify that as part of the branding.

3. Under related repositories, added the verbiage, "This project is part of the Amplify Framework, which runs on Android, iOS, and numerous JavaScript-based web platforms." Added a longer-form section with links to the various projects.